### PR TITLE
feat: arbitration contracts

### PR DESCRIPTION
**Pull request by samb-local**

### CODE REVIEW

File: contracts/contracts/trade/src/contract.rs [hunk 1]
```diff
@@ -676,49 +626,266 @@ fn refund_escrow(
...
+//region arbitration queries
+pub fn query_arbitrator(deps: Deps, arbitrator: Addr) -> StdResult<Vec<Arbitrator>> {
+    let storage = deps.storage;
+
+    let result = arbitrators()
+        .idx
+        .arbitrator
+        .prefix(arbitrator)
+        .range(storage, None, None, Order::Descending)
+        .take(10)
```
Comment (davirds) [inline contracts/contracts/trade/src/contract.rs:819]: Question: 
Why are we returning only 10 items in this query? Shouldn't it be a pagination parameter or return everything if we think it doesn't need pagination?

File: contracts/contracts/trade/src/contract.rs [hunk 2]
```diff
@@ -676,49 +626,266 @@ fn refund_escrow(
...
+        });
     }
+    TradeModel::store(deps.storage, &trade).unwrap();
+
     // Pay arbitration fee
-    if arbitration_mode {
-        let fee_rate: Uint128 = Uint128::new(10);
-        let fee_amount = amount.multiply_ratio(Uint128::new(1), fee_rate);
+    let amount = trade.amount.clone();
+    let fee_rate: Uint128 = Uint128::new(10);
```
Comment (davirds) [inline contracts/contracts/trade/src/contract.rs:787]: Question:
Shouldn't the arbitration fee rate be a parameter?

**Review by davirds — APPROVED** [review]
Everything looks great, only a couple of questions and an observation :D

File: app/src/types/components.interface.ts [hunk 3]
```diff
@@ -147,3 +147,13 @@ export interface UserWallet {
   isConnected: boolean
   address: string
 }
+
+export interface Arbitrator {
+  arbitrator: string
+  fiat: FiatCurrency
+}
+
+export interface SettleDispute {
```
[the remark this answers is not in the document]
Comment (samb-local) [reply app/src/types/components.interface.ts:156]: Good catch!

File: contracts/contracts/trade/src/contract.rs (hunk 1 above)
Comment (samb-local) [reply contracts/contracts/trade/src/contract.rs:819]: Agreed, we can remove the limit for now.

File: app/src/types/components.interface.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (samb-local) [reply app/src/types/components.interface.ts]: Addressed in https://github.com/Local-Money/localmoney/pull/80/commits/c3247b03c182cac9116322c32ffd594eff599898

File: contracts/contracts/trade/src/contract.rs (hunk 1 above)
Comment (samb-local) [reply contracts/contracts/trade/src/contract.rs]: Addressed in https://github.com/Local-Money/localmoney/pull/80/commits/c3247b03c182cac9116322c32ffd594eff599898

File: contracts/contracts/trade/src/contract.rs (hunk 2 above)
Comment (samb-local) [reply contracts/contracts/trade/src/contract.rs]: Addressed in https://github.com/Local-Money/localmoney/pull/80/commits/c3247b03c182cac9116322c32ffd594eff599898

**Review by davirds — APPROVED** [review]

**Pull request merged**